### DISCUSSION TITLE
Listen to a music playlist

### DISCUSSION THREAD

**Discussion by Bestfast** [Syncplay/syncplay#661] [Q&A] [answered]
Is there a way to listen to a music playlist on a Syncplay room, without having every user to switch between files manually? Syncplay doesn't have a built in next/skip function. Maybe there's a hackish way to get around this?

**Comment by Et0h** [comment, marked as answer]
Syncplay has a feature called shared playlists that is media agnostic, and I have used it to play locally stored music files via mpv on Syncplay in the past.

If you want to skip to the next file in the shared playlist you can do so through one of the following methods:
1. Double click the next item in the playlist;
1. Type /qn in the chat box; or
2. Type /qn in the or you can type it in the mpv/mpv.net chat feature

See https://syncplay.pl/guide/client/ for a guide on how to use Syncplay, including the section entitled "Commands accessible via the chat facility".
